R/groupComparisonQCPlots.R

```diff
@@ -190,3 +204,65 @@ modelBasedQCPlots = function(
         dev.off()
     }
 }
+
+
+### new function which is a wrapper for modelBasedQCPlots
+#' Visualization for model-based quality control in fitting model
```

Comment:
Can you run ``devtools::document()`` in Rstudio and commit the extra files generated?  Also if it would be helpful for future developers, we could update the [PR template](https://github.com/Vitek-Lab/MSstats/blob/devel/.github/pull_request_template.md) to remember to run `devtools::document()`